### PR TITLE
Feat/git actions for subgraph creation

### PR DESCRIPTION
**Pull request by estfe**
This PR is related to ticket https://www.notion.so/bd3d552ab0744cde8acba0b65cd32395?v=dd722ce3672f457481c35514729059a9&p=8bd8b5e13dd847638c858a732d1f47bb&pm=s.

It creates a github action to create a subgraph automatically, based on subgraph name that is defined in folder.yml.

### CODE REVIEW

File: .github/workflows/create-subgraph.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      run: |
+        sudo add-apt-repository ppa:rmescandon/yq
+        sudo apt update
+        sudo apt install yq -y
+        yq --version
+
+    - name: Read sub.folder.name from folder.yaml and create subgraph
+      run: |
+        FOLDER_NAME=$(yq eval '.sub.folder.name' folder.yaml)
+        graph create --node http://graph-indexer-prod.airstack.xyz/$FOLDER_NAME
```
Comment (0xsarvesh) [inline .github/workflows/create-subgraph.yml:27]: Please take these endpoints from github secrets.

Comment (0xsarvesh) [reply .github/workflows/create-subgraph.yml:27]: Also you should accept both subgraph name & folder name.

File: .github/workflows/push-subraph.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        cd $FOLDER_NAME
+        npm i
+        npm run deploy
+        npm build
+
+    - name: Read sub.folder.name from folder.yaml and run graph deploy
+      run: |
+        FOLDER_NAME=$(yq eval '.sub.folder.name' folder.yaml)
+        cd $FOLDER_NAME
+        graph deploy --node ${{ secrets.INFURA_NODE }}"
```
Comment (0xsarvesh) [inline .github/workflows/push-subraph.yml:32]: This looks wrong, 
graph deploy --node <> --ipfs <>  <subgraph name>

**Review by 0xsarvesh — CHANGES_REQUESTED** [review]

File: .github/workflows/create-subgraph.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - name: Install yq
+      run: |
+        sudo add-apt-repository ppa:rmescandon/yq
+        sudo apt update
+        sudo apt install yq -y
+        yq --version
+
+    - name: Read sub.folder.name from folder.yaml and create subgraph
+      run: |
+        FOLDER_NAME=$(yq eval '.sub.folder.name' folder.yaml)
```
[the remark this answers is not in the document]
Comment (estfe) [reply .github/workflows/create-subgraph.yml]: Fixed by creating a new env $SUBGRAPH_NAME

File: .github/workflows/push-subraph.yml (hunk 2 above)
Comment (estfe) [reply .github/workflows/push-subraph.yml]: Fixed!

**Review by 0xsarvesh — APPROVED** [review]

**Pull request merged**